global_migrate.go

```diff
@@ -36,17 +36,30 @@ func internalRegister(up, down MigrationFunc, skip int) error {
 //
 // 	package migrations
 // 	import (
-// 		"github.com/globalsign/mgo"
+//	    "go.mongodb.org/mongo-driver/bson"
```

Comment:
I think these imports should be aligned from left

Reply:
updated the doc.